src/Admin/SiteHealth.php

```diff
@@ -267,6 +291,26 @@ public function add_debug_information( $debugging_information ) {
 							'value'   => $this->get_serve_all_templates(),
 							'private' => false,
 						],
+						'amp_css_transient_caching_disabled' => [
+							'label'   => esc_html__( 'Transient caching for stylesheets disabled', 'amp' ),
+							'value'   => $this->get_css_transient_caching_disabled(),
+							'private' => false,
+						],
+						'amp_css_transient_caching_threshold' => [
+							'label'   => esc_html__( 'Threshold for monitoring stylesheet caching', 'amp' ),
+							'value'   => $this->get_css_transient_caching_threshold(),
+							'private' => false,
+						],
+						'amp_css_transient_caching_sampling_range' => [
+							'label'   => esc_html__( 'Sampling range for monitoring stylesheet caching', 'amp' ),
+							'value'   => $this->get_css_transient_caching_sampling_range(),
+							'private' => false,
+						],
+						'amp_css_transient_caching_transient_count' => [
+							'label'   => esc_html__( 'Number of stylesheet transient cache entries', 'amp' ),
+							'value'   => MonitorCssTransientCaching::query_css_transient_count(),
+							'private' => false,
+						],
```

Comment:
Should not some of these be omitted if `wp_using_ext_object_cache()` and thus transients are not being used?

Reply:
I'd keep them around. If people have trouble with transient caching and switch to the object cache just to diagnose, we'd still want to have the values to help with debugging.

In general, you'd want to make as few assumptions as possible when collecting debugging information, as you'll make the same assumptions you've already done when coding up the bug in the first place.